modules/express/README.md

```diff
@@ -163,31 +163,48 @@ BitGo Express currently supports the following proxy protocols:
 * SOCKSv5
 * PAC
 
+### External Signing Mode
```

Comment:
Let's add some text here to say that this mode is currently available only in testnet and is under active development. Breaking changes may be made to it without incrementing the major version until the feature is stabilized and made available in production.